salsa_dancing_molecules/force.py

```diff
@@ -0,0 +1,16 @@
+"""Functionfile for forces.
+
+Function to get forces from a ASE atoms object.
```

Comment:
I think this description would be better if put in the actual get_force function instead of the module.